### PR TITLE
Add fundrawtransaction command, fix signing commands

### PR DESCRIPTION
**Pull request by stevenroose**
If this is merged, I will release v0.5.0, since this breaks existing interface.

### CODE REVIEW

File: client/src/client.rs [hunk 1]
```diff
@@ -57,6 +57,16 @@ fn null() -> serde_json::Value {
     serde_json::Value::Null
 }
 
+/// Shorthand for an empty serde_json::Value array.
+fn empty() -> serde_json::Value {
```
Comment (dpc) [inline client/src/client.rs]: `empty_arr` or `empty_vec`?

**Comment by stevenroose** [comment]
You're right :D I just copied this from liquid-rpc where `empty_obj` didn't exist yet. I'll change it to `empty_arr`.

**Comment by dpc** [comment]
LGTM. We should eventually work on the `test_against_node.rs` or something like this, to be able to run it in travis against regtest node, and see if all the calls are still working. Nothing fancy, just some sanity.

**Comment by stevenroose** [comment]
Yeah totally agree. There is an issue for this too.

**Pull request merged**